ajax/libs/tsparticles/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "tsparticles",
+  "version": "1.6.0",
+  "description": "Porting of Vincent Garreau's particles.js, converted in TypeScript.",
+  "homepage": "https://github.com/matteobruni/tsparticles",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/matteobruni/tsparticles.git"
+  },
+  "keywords": [
+    "particles",
+    "particle",
+    "canvas",
+    "particles-js",
+    "particles.js",
+    "typescript",
+    "javascript",
+    "animation"
+  ],
+  "npmName": "tsparticles",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "author": "Matteo Bruni",
+  "license": "MIT",
+  "main": "dist/tsparticles.min.js"
```

```suggestion
  "filename": "dist/tsparticles.min.js"
```
We use the `filename` property for the default file, not `main`.